test/libyul/YulOptimizerTestCommon.cpp

```diff
@@ -311,7 +311,7 @@ YulOptimizerTestCommon::YulOptimizerTestCommon(
 		}},
 		{"fullSuite", [&]() {
 			GasMeter meter(dynamic_cast<EVMDialect const&>(*m_dialect), false, 200);
-			OptimiserSuite::run(*m_dialect, &meter, *m_object, true, solidity::frontend::OptimiserSettings::DefaultYulOptimiserSteps);
+			OptimiserSuite::run(*m_dialect, &meter, *m_object, true, solidity::frontend::OptimiserSettings::DefaultYulOptimiserSteps, 200);
```

Comment:
Does `runs = 200` pass this check i.e., enable the optimisation in this PR?

https://github.com/ethereum/solidity/pull/11026/files#diff-f9c9dce78307a3b319b2c54b0baf3911125ccd830ebfca1a45ce44eab92a07fdR133

Reply:
Looks like it, just cross-checking.